seacrowd/sea_datasets/lex_indo/lex_indo.py

```diff
@@ -0,0 +1,100 @@
+from pathlib import Path
+from typing import List
+
+import datasets
+
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses
+
+_CITATION = """\
+@misc{magichubLEXIndoIndonesian,
+  author    = {},
+  title     = {LEX-INDO: AN INDONESIAN LEXICON},
+  year      = {},
+  howpublished = {Online},
+  url       = {https://magichub.com/datasets/indonesian-lexicon/},
+  note      = {Accessed 19-03-2024},
+}
+"""
+
+_DATASETNAME = "lex_indo"
+
+_DESCRIPTION = """This open-source lexicon consists of 2,000 common Indonesian words, with phoneme series attached.
+It is intended to be used as the lexicon for an automatic speech recognition system or a text-to-speech system.
+The dictionary presents words as well as their pronunciation transcribed with an ARPABET(phone set of CMU)-like phone set. Syllables are separated with dots.
+"""
+
+_HOMEPAGE = "https://magichub.com/datasets/indonesian-lexicon/"
+_LICENSE = Licenses.CC_BY_NC_ND_4_0.value
+_LOCAL = True
+
+_URLS = {}
+
+_SUPPORTED_TASKS = []
+
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+
+class LexIndo(datasets.GeneratorBasedBuilder):
+    """This open-source lexicon consists of 2,000 common Indonesian words, with phoneme series attached"""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",
+            version=datasets.Version(_SOURCE_VERSION),
+            description="lex_indo lexicon with source schema",
+            schema="source",
+            subset_id="lex_indo",
+        )
+    ]
```

Comment:
There should be `seacrowd` schema instead of only `source`, especially since `Tasks.MULTILEXNORM` has been supported by `T2T` schema.